src/components/admin/MainIssueController.jsx

```diff
@@ -156,8 +156,26 @@ export default class MainIssueController extends FalcorController {
               return false;
             }
           }
+          if (/<a.*?href\s*?=\s*?["'](?!http).*?>/.test(article.html)) {
+            if (!window.confirm(
+              `${article.title} is in non-absolute format,
+              which means that it does not have http(s):// in front of it. ` +
+              'This problem will cause the link to break in the' +
+              'browser due to something called relative links.' +
+              'It will be misinterpreted and it will simply add the' +
+              'link written to the end of the URL.' +
```

Comment:
All these places where you don't have space at the end of the last string or the start of the next string will be concatenated as such: `...will simply add thelink written to...`